index.html

```diff
@@ -10,35 +10,27 @@
 </head>
 <body>
```

Comment:
## Please watch this video before you begin changes. 
https://www.loom.com/share/adad94e9dd4e4e90ab55ea846c19f533